CONTRIBUTING.md

```diff
@@ -1,270 +1,54 @@
-# Contributing to Hack Club
+# How To Contribute
 
-Contributions to Hack Club are very welcome, and strongly encouraged (even to
-this document)!
+This is a guide explaining the contribution workflow we use at Hack Club!
 
-## Doing it
+- Find something you want to change
```

Comment:
I think these bullet points should be internal links to their associated headings. For example

```
Find something you want to change
```

can be changed to

```md
[Find something you want to change](#find-something-to-change)
```